examples/agentic/mcp_distillation_evaluation/.env.example

```
@@ -0,0 +1,21 @@
+
+# ── Required ──────────────────────────────────────────────────────────
+OPENAI_API_KEY=sk-...
+
+# ── Model configuration (optional — defaults shown) ──────────────────
+# TEACHER_MODEL=openai/gpt-5.2
+# JUDGE_MODEL=openai/gpt-4o
+
+# ── Only needed for task generation (Section 2) ──────────────────────
+# Skip these if using the pre-generated tasks in outputs/
+LANGFLOW_URL_WEATHER_DATA=http://localhost:7860/api/v1/run/<flow-id>
+LANGFLOW_URL_MEDICAL_CALCULATOR=http://localhost:7860/api/v1/run/<flow-id>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LANGFLOW_URL_MEDICAL_CALCULATOR key should go before the LANGFLOW_URL_WEATHER_DATA key

(UnorderedKey)